recipes/pcaone/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.7" %}
+{% set sha256 = "e914dc2d3e1c64d027dd1d1b9338e4fc7789a647f7124cec717480dd0c584c25" %}
+
+
+package:
+  name: pcaone
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/Zilong-Li/PCAone/archive/refs/tags/v{{ version }}.tar.gz # [x86_64]
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
The skip line can be removed, Bioconda doesn't have Windows builds.

Reply:
ooh, removed! just curious what are the reasons of not supporting Windows

Reply:
I assume the original rationale was that most users are on Linux or macOS, so native Windows builds are unnecessary overhead. These days Windows also has the Windows Subsystem for Linux which lets you execute native Linux binaries, so there's little reason for native Windows builds.

Reply:
Thank you very much! Have a nice day!